apps/client/src/common/components/view-params-editor/constants.ts

```diff
@@ -224,6 +230,12 @@ export const PUBLIC_OPTIONS: ParamField[] = [
     description: 'Schedule will not auto-cycle through events',
     type: 'boolean',
   },
+  {
+    id: 'eventsPerPage',
+    title: 'Number of events on page',
+    description: '',
```

Comment:
Just for sake of consistency we can use the title and description fields
The description field can also warn users that the events will overflow the page